docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+
+### Privileges
+
+Roles are explicitly granted privileges to access objects in the database using the `GRANT` command. You can, for example, assign read access to one role, data modify access to another role, and alter table access to a third.
+
+By default, when an object is created, only the owner has privileges, and other roles must be granted privileges.
+
+### Row-level access
+
+In addition to database access permissions available through the `ROLE` and `GRANT` privilege system, YugabyteDB provides a more granular level of security where tables can have row security policies that restrict the rows that users can access.
+
+Row-level Security (RLS) restricts rows that can be returned by normal queries or inserted, updated, or deleted by DML commands. Row-level security policies can be created specific to a `DML` command or with `ALL` commands. They can also be used to create policies on a particular role or multiple roles.
+
+By default, tables do not have any row level policies defined, so that if a user has access privileges to a table, all rows within the table are available to query and update.
+
+### Column-level access
+
+Column-level security in YugabyteDB is used to restrict users to viewing only a particular column or set of columns in a table. You do this by creating a view that includes only the columns that the user needs access to using the `CREATE VIEW` command.
+
+## Encryption
+
+YugabyteDB supports encryption both of clusters, and all network communication between servers. 
+
+### Encryption in transit
+
+[TLS encryption](https://en.wikipedia.org/wiki/Transport_Layer_Security) ensures that network communication between servers is secure. You can configure YugabyteDB to use TLS to encrypt intra-cluster and client to server network communication. Servers are secured using TLS certificates, which can be from a public CA or self-signed.
+
+You should enable encryption in transit for YugabyteDB clusters and clients to ensure privacy and integrity of data transferred over the network.
+
+Read more about enabling [Encryption in transit](../../secure/tls-encryption) in YugabyteDB.
+
+### Encryption at rest
+
+[Encryption at rest](https://en.wikipedia.org/wiki/Data_at_rest#Encryption) ensures that data at rest, i.e., stored on disk, is protected. You can configure YugabyteDB with a user-generated symmetric key to perform cluster-wide encryption.
+
+Read more about enabling [Encryption at rest](../../secure/encryption-at-rest) in YugabyteDB.
+
+### Column-level encryption
```

Comment:
You forgot this one in the introductory paragraph.

Reply:
fixed